cmd/agent/app/builder_test.go

```diff
@@ -55,7 +55,7 @@ processors:
         maxPacketSize: 65001
         hostPort: 3.3.3.3:6832
 
-samplingServer:
+httpServer:
```

Comment:
internally we've already updated puppet to use httpServer, where else could it break?